### PR TITLE
Export CloseButton for public use & Fix example docs error

### PR DESCRIPTION
**Pull request by BrennerSpear**
Comment #2623 

Includes tests

### CODE REVIEW

**Comment by BrennerSpear** [comment]
What are y'alls thoughts on defaulting the label to "Close" or "Close Item"?

**Comment by jquense** [comment]
I fee good about "Close" as a default yeah

**Comment by BrennerSpear** [comment]
Will add that. Should it be on the same PR or a new one?

**Comment by BrennerSpear** [comment]
Newest commit fixes issue #2338.

Nothing wrong with the codebase - just the example. and ```id``` was getting passed into ```NavDropdown``` when it shouldn't have been.

I can put these into two different PRs if you'd like me to, but I think this commit is small enough to just tack on.

File: test/CloseButton.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
```
Comment (taion) [inline test/CloseButton.js]: should be `react-dom/test-utils`

Comment (BrennerSpear) [inline test/CloseButton.js]: Oops. Fixed

**Pull request merged**